packages/fxa-auth-server/lib/routes/subscriptions/stripe.ts

```diff
@@ -441,15 +441,28 @@ export class StripeHandler {
       !customer || customer.tax?.automatic_tax === 'supported';
     const automaticTax = this.automaticTax && isCustomerTaxed;
 
-    const previewInvoice = await this.stripeHelper.previewInvoice({
-      automaticTax,
-      country,
-      ipAddress,
-      promotionCode,
-      priceId,
-    });
+    try {
+      const previewInvoice = await this.stripeHelper.previewInvoice({
+        automaticTax,
+        country,
+        ipAddress,
+        promotionCode,
+        priceId,
+      });
 
-    return stripeInvoiceToFirstInvoicePreviewDTO(previewInvoice);
+      return stripeInvoiceToFirstInvoicePreviewDTO(previewInvoice);
+    } catch (err: any) {
+      if (err.type === 'StripeInvalidRequestError') {
```

Comment:
What's the purpose of the `INVALID_INVOICE_PREVIEW_REQUEST` defined previously if not for this type of check?

Reply:
Answered via slack, thanks!